pages/use-cases/security/security-baseline.mdx

```diff
@@ -0,0 +1,141 @@
+- Using Scaleway Load Balancers in front of your Instances, with an Edge Services pipeline providing a WAF. This protects your Instances from DDoS attacks as well as other categories of attack.
+- Using Scaleway Security Groups to restrict inbound traffic only to necessary ports, and avoid exposing services like SSH to the public internet.
+- Regularly monitoring your traffic using Scaleway Cockpit, and set up alerts for unusual traffic spikes.
+- Remove public IPs from your Instances wherever possible, and enable communication over Private Networks and VPCs.
+
+## Identity and Access Management
+
+Effective access control ensures that only authorized users and systems can interact with your Scaleway infrastructure. This section covers the core principles of Scaleway IAM. By properly configuring users, applications, API keys and permissions, you can prevent unauthorized actions, reduce the risk of credential misuse, and maintain clear accountability across your environment.
+
+### Least privilege by design: secure access with granular permissions
+
+If you want to share access to your Scaleway Organization, then invite other users as Members. You can then **accord fine-grained permissions to each Member**, via IAM policies. Policies determine which permissions and access rights the Member has, e.g. to manage billing, create different types of resources, read-only certain types of resources, and more.
+
+You may also want to give access to your Organization and resources not to a specific human user, but to an application or service, e.g. when setting up a production environment. Do this by creating IAM applications. This feature lets you **give programmatic access to resources** by creating API keys that are not linked to a specific human, making your production code more robust.
+
+We always recommend that you **give least-privilege permissions** via IAM, as best practice. This means giving users and applications to permissions they need to perform a task, and no extra permissions beyond that. The Organization Owner has powerful permissions over the whole Organization that go far beyond the permission scope required for most tasks. Even if you do not need to invite additional Members to your Organization, consider creating additional applications whose API keys have least-privilege-only permissions. You can then use these API keys in your applications without exposing the Organization Owner's API key with its full permissions.
+
+Find out more:
+
+- [IAM Quickstart](/iam/quickstart/)
+- [Understanding IAM](/iam/reference-content/overview/)
+
+### Verify identity: require MFA for secure account access
+
+We recommend setting up **M**ulti**f**actor **A**uthentication (MFA) to add an extra layer of security to your Scaleway account. When MFA is enabled, you are prompted to provide a second security measure, in addition to your password, when logging in. This could be via a one-time password accessed via an app, or a secure passkey on your device.
```

Comment:
there's two mfa methods currently available: https://www.scaleway.com/en/docs/account/how-to/use-2fa/

- One-time password (TOTP) - method through which you access your account using a unique, time-based validation code generated by an app.
- Passkey - secure authentication method that eliminates the need to create, manage, or remember passwords. (ONLY AVAILABLE FOR OWNERS)